app/test_main.py

```diff
@@ -1,3 +1,40 @@
 from app.main import get_human_age
+import pytest
 
-# write your code here
+
+class TestDogAge:
```

Comment:
why did you decide to put these tests in a class?

Reply:
cause i think if we will make more tests for this func , separate classes is right 
It's more comfortable for me)